<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>control_panel::capture::InputResolution</class>
 <widget class="QWidget" name="ControlPanel_Capture_InputResolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>349</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Force resolution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBox_customWidth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>700</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_customHeight">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>700</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_applyCustomResolution">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_inputForceButtons">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_inputForce_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>720</width>
             <height>400</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_inputForce_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>1024</width>
             <height>768</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_inputForce_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_inputForce_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>400</width>
             <height>300</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_inputForce_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>512</width>
             <height>384</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_inputForce_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>800</width>
             <height>600</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_inputForce_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>640</width>
             <height>400</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_inputForce_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>640</width>
             <height>480</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_inputForce_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>[&quot;resolution&quot;]</string>
           </property>
           <property name="resolution" stdset="0">
            <size>
             <width>1280</width>
             <height>1024</height>
            </size>
           </property>
           <property name="button_idx" stdset="0">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_inputForce_7</tabstop>
  <tabstop>pushButton_inputForce_8</tabstop>
  <tabstop>pushButton_inputForce_9</tabstop>
  <tabstop>pushButton_inputForce_4</tabstop>
  <tabstop>pushButton_inputForce_5</tabstop>
  <tabstop>pushButton_inputForce_6</tabstop>
  <tabstop>pushButton_inputForce_1</tabstop>
  <tabstop>pushButton_inputForce_2</tabstop>
  <tabstop>pushButton_inputForce_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
